tests/ember/EmberTemplateLintBearTest.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+from bears.ember.EmberTemplateLintBear import EmberTemplateLintBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+good_file = """
+<body>
+  body: {{{body}}}
+  <hr>
+  post body
+  <div style="background-image: url({{url}})"></div>
+</body>
+"""
+
+bad_file = """
+<body>
+  body: {{{body}}}
+  <hr/>
+  post body
+  <div style="background-image: url({{url}})">
+</body>
+"""
+
+good_file_config = """
```

Comment:
The name is a bit confusing.

Reply:
yes you are write, this file denotes a good file when a config file is used. Is `good_file_with_config`will be good enough?

Reply:
Yeah, that would be better